manifests/init.pp

```diff
@@ -5,6 +5,7 @@
 
   Enum['absent', 'latest', 'present', 'purged'] $package_ensure = 'present',
   String[1] $package_name = 'fail2ban',
+  Array $package_install_options = [],
```

Comment:
```suggestion
  Array[String[1]] $package_install_options = [],
```

Reply:
I think it is better to use the allowed data types mentioned in https://puppet.com/docs/puppet/7/types/package.html#package-attribute-install_options than to specify that it requires at least one install_option.

I've also added a spec test that resembles the one used in another Voxpupuli module: <https://github.com/voxpupuli/puppet-collectd/blob/60d656ab1063b30f7617caf6e4115cc0050ca2eb/spec/classes/collectd_plugin_sensors.rb#L109>

Reply:
Ah yes, that's good.